src/nitin/WhileLoop.java

```diff
@@ -0,0 +1,55 @@
+package nitin;
+
+public class WhileLoop {
+	
+	void evenNumber(int start,int end) {
+		int index = start;
```

Comment:
why do we need extra variable here?
you can use start directly as increment variable.

Reply:
Thank you Harsh for your comment, I will try it based on your suggestion, do i have to submit the assignment again in the same pull request?

Reply:
Thank you i have tried again and it worked, i am pushing it as a practice assignment.